cloudflared/config.yaml

```diff
@@ -20,11 +20,12 @@ options:
   tunnel_name: "homeassistant"
   additional_hosts: []
 schema:
-  external_hostname: str
-  tunnel_name: str
+  external_hostname: str?
```

Comment:
I like the suggestion to make the two variables `external_hostname` and `tunnel_name` optional as well. Nevertheless, now we do not have sufficient checks in place, in case the user starts the add-on without those two variables but also without setting `quick_tunnel`.
I think a good solution would be to check in `cloudflared-config.sh` after checking that `quick_tunnel` is zero (line 237), to also check if `external_hostname` and `tunnel_name` are set. Otherwise, we can abort with an error stating that you either have to set those two or `quick_tunnel`.
What do you think?

Reply:
Makes sense! I've made the change 👍